generators/server/generator.mjs

```diff
@@ -351,7 +351,11 @@ class `
             )
             .replaceAll(
               `@RequestParam(required = false, defaultValue = "false") boolean eagerload`,
-              `@RequestParam(name = "eagerload",required = false, defaultValue = "false") boolean eagerload`
+              `@RequestParam(name = "eagerload", required = false, defaultValue = "true") boolean eagerload`
```

Comment:
@mshima Why do we need to replace `defaultValue = "false"` with `defaultValue = "true"`? If the default should be true, shouldn't that be done in JHipster core?

Reply:
Yes I will do that.

Reply:
I think you are right, let’s keep the same behavior.
```suggestion
              `@RequestParam(name = "eagerload", required = false, defaultValue = "false") boolean eagerload`
```